Formula/d/dosbox-x.rb

```diff
@@ -67,7 +67,7 @@ def install
     ]
 
     system "./autogen.sh"
-    system "./configure", *args, *std_configure_args
+    system "./configure", *args, *(std_configure_args - ["--disable-debug"])
```

Comment:
It's better to remove `std_configure_args` and add only `--prefix=#{prefix}` to the `args`.

```
    args = %W[
      --prefix=#{prefix}
      --enable-debug=heavy
      ...
    ]
```

Reply:
I've reverted to the original approach. I also like it better as it doesn't get rid of `libdir` (computed [here](https://github.com/Homebrew/brew/blob/4d14be89e99a45181c18e96a5f19a5b43343cc0f/Library/Homebrew/formula.rb#L1811), so not a simple/interpolated string). `dosbox-x` doesn't put anything under `libdir`, but it may in the future... Removing `--disable-debug` from `std_configure_args`, instead of hardcoding everything in `args`, seems to be more future-proof/more general for other formulas to follow.

Reply:
Please use `*std_configure_args.reject { |s| s["--disable-debug"] }`